src/frontend/src/pages/HomePage/OnboardingHomePage.tsx

```diff
@@ -22,10 +23,14 @@ const OnboardingHomePage = () => {
 
   return (
     <PageLayout title="Home" hidePageTitle>
-      <Grid display={'flex'} alignItems={'center'} marginLeft={2} marginTop={4}>
-        <Typography sx={{ fontSize: '2.5em' }}>Welcome to the Northeastern Electric Racing Team</Typography>
+      <Grid container display={'flex'} alignItems={'center'} marginLeft={2} marginTop={4}>
+        <Grid item xs={12} md={7}>
+          <Typography sx={{ fontSize: '2.5em' }}>Welcome to the Northeastern Electric Racing Team</Typography>
```

Comment:
This shouldn’t be hardcoded, this should be the organizations title

Reply:
I will make another ticket for this becayse an organization doesn't have a titleField right now

Reply:
use organization name